frontend/src/v5/ui/controls/dragAndDrop/dragAndDrop.component.tsx

```diff
@@ -15,43 +15,22 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { useState } from 'react';
-import { Container, DashedContainer, DropArea } from './dragAndDrop.styles';
+import { DashedContainer, DropArea } from './dragAndDrop.styles';
+import { useDropzone } from 'react-dropzone';
 
 interface IDragAndDrop {
-	hidden?: boolean;
```

Comment:
This was used to hide the dragAndDrop container. If you want to remove it, you may need to adjust the components that were passing it (I can only think of of ticket comments creation and after uploading a new revision file, but there may be others)

e.g. container revision upload after uploading:

staging (after uploading)
![image](https://github.com/user-attachments/assets/bd8242bb-f9a8-46a9-a0c3-930a89e17599)


now (after uploading)
![image](https://github.com/user-attachments/assets/b9e551a8-b33a-484c-8438-e0172c96f6ec)